src/service/command-executor.js

```diff
@@ -44,6 +47,36 @@ function testCopyJUnitReports(outputDir) {
   files.forEach((file) => fs.copyFileSync(path.join(sampleDir, file), path.join(outputDir, file)));
 }
 
+function checkFileExist(ksProjectPath, newTestSuitePath) {
```

Comment:
move this logic to file.js
The function checks if the path exists or not. It is not  specific to TestSuite